carma_wm/include/carma_wm/WMListener.h

```diff
@@ -0,0 +1,106 @@
+#pragma once
```

Comment:
Do we have a preference for `#pragma once` vs `#ifndef __...___` guards?

Reply:
Shuwei and I have been using ```#pragma once``` but I don't have a big preference either way. Technically if we tries to compile this for something other than ubuntu it could cause issues if the compiler did not support pragma, but that seems unlikely at the moment.